front_end/audits2/Audits2Panel.js

```diff
@@ -1,43 +1,242 @@
 // Copyright (c) 2016 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+
+/**
+ * @typedef {{
+ *     lighthouseVersion: ?string,
```

Comment:
will this ever be `null` or will it be `undefined`? If undefined you have to do the more unfortunate syntax of `string|undefined`